nixos-modules/host/options.nix

```diff
@@ -37,9 +37,8 @@
         ''${pkgs.iproute2}/bin/ip link set "$id" master br0 up
       '';
       type = types.lines;
-      default = ''
-        ${pkgs.iproute2}/bin/ip link set "$id" up
-      '';
+      default = "${pkgs.iproute2}/bin/ip link set \"$id\" up";
```

Comment:
Can we keep the `default = ''` with a newline, now that we have the `defaultText` too?